test/test_helper.exs

```diff
@@ -1,2 +1,23 @@
 ExUnit.configure(exclude: :external)
 ExUnit.start()
+
+defmodule PrimaAuth0Ex.TestHelper do
+  import ExUnit.Callbacks
+
+  @spec set_client_env(term(), term(), boolean()) :: :ok
+  def set_client_env(key, value, false) do
+    Application.get_env(:prima_auth0_ex, :client, [])
+    |> Keyword.put(key, value)
+    |> then(&Application.put_env(:prima_auth0_ex, :client, &1))
+  end
+
+  def set_client_env(key, value, true) do
```

Comment:
Could the boolean parameter be a keyword list instead, something like `reset_after_test: true`? By reading the test it isn't clear what it does.

Btw, these helpers are nice: can be used in other tests too?